gammapy/maps/geom.py

```diff
@@ -1063,6 +1063,17 @@ def coord_to_pix(self, coords):
         """
         pass
 
+    def get_mapcoord(self):
+        """Returns a MapCoord for this `~gammapy.maps.Geom` object.
+        """
+        coord_tuple = self.get_coord()
+        cdict = dict(lon=coord_tuple[0], lat=coord_tuple[1])
```

Comment:
I'd suggest instantiating the `MapCoord` with an `OrderedDict` object so that the axis ordering of the map is preserved.